packages/pluggableWidgets/line-chart-native/src/components/LineChart.tsx

```diff
@@ -50,28 +52,90 @@ export function LineChart(props: LineChartProps): ReactElement | null {
 
             const seriesStyle = style.series && stylePropertyName ? style.series[stylePropertyName] : undefined;
 
+            const displayMarker = seriesStyle?.markers?.display;
+
+            if (
+                displayMarker !== undefined &&
+                !(displayMarker === "false" || displayMarker === "underneath" || displayMarker === "onTop")
+            ) {
+                console.warn(
+                    `${
+                        warningPrefix ? warningPrefix + "i" : "I"
+                    }nvalid value for series marker style property, display, valid values are "false", "underneath" and "onTop".`
+                );
+            }
+
             const markers =
                 lineStyle === "lineWithMarkers" ||
-                (lineStyle === "custom" && seriesStyle?.markers?.display !== "false") ? (
+                (lineStyle === "custom" && seriesStyle?.markers?.display && seriesStyle.markers.display !== "false") ? (
                     <VictoryScatter data={dataPoints} style={seriesStyle?.markers} size={seriesStyle?.markers?.size} />
                 ) : (
                     undefined
                 );
 
             return (
                 <VictoryGroup key={index}>
-                    {markers && (!seriesStyle?.markers?.display || seriesStyle.markers.display !== "onTop")
+                    {markers && seriesStyle?.markers?.display === "underneath" ? markers : null}
+                    <VictoryLine style={seriesStyle?.line} data={dataPoints} interpolation={interpolation} />
+                    {markers && (!seriesStyle?.markers?.display || seriesStyle?.markers?.display !== "underneath")
                         ? markers
                         : null}
-                    <VictoryLine style={seriesStyle?.line} data={dataPoints} interpolation={interpolation} />
-                    {markers && seriesStyle?.markers?.display === "onTop" ? markers : null}
                 </VictoryGroup>
             );
         });
     }, [dataTypesResult, series, style]);
 
     const [firstSeries] = series;
 
+    const axisLabelStyles = useMemo(() => {
+        const [extractedXAxisLabelStyle, xAxisLabelStyle] = extractStyles(style.xAxisLabel, ["relativePositionGrid"]);
+        const [extractedYAxisLabelStyle, yAxisLabelStyle] = extractStyles(style.yAxisLabel, ["relativePositionGrid"]);
+
+        if (
+            !(
+                extractedXAxisLabelStyle.relativePositionGrid === "bottom" ||
+                extractedXAxisLabelStyle.relativePositionGrid === "right"
+            )
+        ) {
+            if (extractedXAxisLabelStyle.relativePositionGrid !== undefined) {
+                console.warn(
+                    `${
+                        warningPrefix ? warningPrefix + "i" : "I"
```

Comment:
Perhaps make this a constant and reference this throughout.

Reply:
Changed it